src/cpu/core_dynrec/dyn_fpu.h

```diff
@@ -387,11 +389,12 @@ static void dyn_fpu_esc3(){
 //				LOG(LOG_FPU,LOG_ERROR)("80267 protected mode (un)set. Nothing done");
 				break;
 			default:
-				E_Exit("ESC 3:ILLEGAL OPCODE group %d subfunction %d",decode.modrm.reg,decode.modrm.rm);
+				E_Exit("ESC 3:ILLEGAL OPCODE group %" PRIuPTR " subfunction %" PRIuPTR,
+				       decode.modrm.reg, decode.modrm.rm);
 			}
 			break;
 		default:
-			LOG(LOG_FPU,LOG_WARN)("ESC 3:Unhandled group %d subfunction %d",decode.modrm.reg,decode.modrm.rm);
+			FPU_LOG_WARN(3, false, decode.modrm.reg, decode.modrm.rm);
```

Comment:
Should the 2nd arg be true like the others below? (Just a question)

Reply:
Nope:
`false` -> `"ESC 3:…"`
`true` -> `"ESC 3 EA:…"`

I have no idea what "EA" means, but that's how upstream designed interface of FPU_LOG_WARN helper function…

Reply:
Ah! Subtle; I didn't catch that - thank for pointing out this difference.